### DISCUSSION TITLE
nlohmann/json

### DISCUSSION THREAD

**Discussion by Garald-48rus** [nlohmann/json#4326] [Q&A]
Translator Good time of the day. I downloaded the Json.tar.zx archive, then unpacked it, but my IDE QT Creator says 'nlohmann/json.hpp' file not found

**Comment by nlohmann** [comment]
This is a project include-path setup issue rather than a library bug. After extracting the release tarball, the header needs to be on the compiler's include search path -- add the extracted include/ directory (which contains the nlohmann/ subfolder) to the QT Creator project's INCLUDEPATH (or use the standalone single_include/nlohmann/json.hpp and add its containing directory). Simply unpacking the archive doesn't put the header on any default search path; see docs/mkdocs/docs/integration/index.md for the supported ways to add the library to a project (single header, CMake, package managers).

---
*This reply was drafted by [Claude Code](https://claude.com/claude-code) on behalf of @nlohmann, as part of a review of open discussions.*
